packages/browser-repl/package.json

```diff
@@ -15,6 +15,7 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006",
     "test": "karma start ./config/karma.conf.js --colors",
+    "test-ci": "echo Not tested in CI",
```

Comment:
wait really? this wasn't tested? I think we should re-add it. Do we need a ticket or would just work?

Reply:
also great you spotted it :)

Reply:
Err, I can change it and see what happens? (Fwiw, I'm not entirely clear on why test and test-ci are seperate scripts anyway... it doesn't usually seem like a good idea to me to have local tests do something different than CI)

Reply:
Opened https://github.com/mongodb-js/mongosh/pull/386 to see if this works

Reply:
> it doesn't usually seem like a good idea to me to have local tests do something different than CI

I'm not sure it is doing something different. If it is, at least till now is not on purpose. Then again if it will be useful for some reason I don't know, feel free to remove it.